homeassistant/components/peco/peco_outage_api.py

```diff
@@ -0,0 +1,86 @@
+"""Main object for getting the PECO outage counter data."""
+import httpx
+
+from .const import API_URL, COUNTY_LIST
+
+
+class PecoOutageApi:
```

Comment:
Does this file setup an API to communicate with an external service? The first item on the [development checklist](https://developers.home-assistant.io/docs/development_checklist/) suggests this is not allowed so you would need to add your API to a separate repo and then publish it on pypi.

Reply:
It seems a bit excessive, but if it's required, I'll do it.

Reply:
Maybe, but its the policy of this project. You could run this as a custom integration published on HACS if you really want everything bundled together.

Reply:
Alright. Just published a library to pypi. Going to implement it into the integration now
It's available here: https://pypi.org/project/peco

Reply:
nice! Consider adding support to optionally pass a websession to it.
see https://developers.home-assistant.io/docs/integration_quality_scale_index#platinum- last point

Reply:
Can you explain how to implement this? I don't see much documentation on the website.

Reply:
I'll just switch to `aiohttp`. Shouldn't be that hard ^.^